test/extended/authorization/rbac/groups_default_rules.go

```diff
@@ -206,7 +206,7 @@ func testGroupRules(ruleResolver validation.AuthorizationRuleResolver, group, na
 	}
 
 	// force test data to be cleaned up every so often but allow extra rules to not deadlock new changes
-	if cover, missing := validation.Covers(actualRules, expectedRules); !cover && len(missing) > 12 {
+	if cover, missing := validation.Covers(actualRules, expectedRules); !cover && len(missing) > 15 {
```

Comment:
You should not need this since the other resources should be in your `manifests` now right?

Reply:
true, i will eliminate the bump in #